containers/alpine/.devcontainer/devcontainer.json

```diff
@@ -5,13 +5,19 @@
 		// Update 'VARIANT' to pick an Alpine version: 3.13, 3.14, 3.15
 		"args": { "VARIANT": "3.15" }
 	},
-	
-	// Set *default* container specific settings.json values on container create. 
-	"settings": {},
 
-	// Add the IDs of extensions you want installed when the container is created.
-	// Note that some extensions may not work in Alpine Linux. See https://aka.ms/vscode-remote/linux.
-	"extensions": [],
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			// Note that some extensions may not work in Alpine Linux. See https://aka.ms/vscode-remote/linux.
```

Comment:
Assuming the note on Alpine only goes into Alpine templates.

Reply:
Yes! Good call out though. I copied over the note on Alpine from the existing devcontainer.json.